client/state/preferences/test/reducer.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import {
+	PREFERENCES_SET,
+	PREFERENCES_RECEIVE,
+	PREFERENCES_FETCH,
+	PREFERENCES_FETCH_SUCCESS,
+	PREFERENCES_FETCH_FAILURE,
+	SERIALIZE,
+	DESERIALIZE
+} from 'state/action-types';
+import reducer, { createReducerForPreferenceKey, fetching } from '../reducer';
+import { DEFAULT_PREFERENCES, USER_SETTING_KEY } from '../constants';
+
+describe( 'reducer', () => {
+	it( 'should export expected reducer keys', () => {
+		expect( reducer( undefined, {} ) ).to.have.keys( [
+			'values',
+			'fetching'
```

Comment:
I'm wondering if we should (and also if we really can) also check for nested `values` keys here.